services/TikTok.filters.js

```diff
@@ -0,0 +1,32 @@
+export function selectOnlyEuropeanTOS(document) {
+  const firstElement = document.querySelector('article > *:first-child')
+  const titleEuropean = document.querySelector('#terms-eea');
+  const titleIndia = document.querySelector('#terms-in')
+  const lastElement = document.querySelector('article > *:last-child')
+
+  const range1 = document.createRange();
```

Comment:
var name like `range1` is not very explicit. Try to explicit the meaning of the variable, for example `otherJuridictionsContent` or something like that.